README.md

```diff
@@ -13,6 +13,11 @@ var path = require("path");
 const config = {
   // path to your assets
   assetsPath: path.join("app", "assets", "images"),
+  // asset gets copied to this folder if provided
+  assetsOutputPath: path.join("public/copies"),
+  // link to assetsOutputPath with hash goes here
+  // this is only useful when you copy the file using assetsOutputPath
```

Comment:
I think there was a miscommunication about the purpose of `assetsLink` My understanding of the value is to not have 2 copies of the file (one named img.png and one named img-59randomhash123.png) so we might just be able to create a symlink for the hashed version. What's the value of having the link here?